### DISCUSSION TITLE
is there any MySqlScript replacement?

### DISCUSSION THREAD

**Discussion by vishnukumartr** [mysql-net/MySqlConnector#1382] [Q&A]
Hello Everyone,

migrating from MySql.Data.MySqlClient to MySqlConnector,

is there any MySqlScript replacement in MySqlConnector?

your help will be much appreciable!! 

Thanks & Regards,
Vishnu Kumar T R

**Comment by bgrainger** [comment]
See response here and other comments on this issue: https://github.com/mysql-net/MySqlConnector/issues/382#issuecomment-340562191

What do you use `MySqlScript` for that you couldn't do with just `MySqlCommand` or a helper library like Dapper?

**Reply by vishnukumartr** [reply]
Thanks for the response @bgrainger ! I will check on this.

Actually having some sqlscript files (contains stored procedures) to execute.
